UI/window-basic-main.cpp

```diff
@@ -6648,7 +6652,10 @@ void OBSBasic::BroadcastButtonClicked()
 					QTStr("Output.BroadcastStartFailed"),
 					last_error, true);
 				ui->broadcastButton->setChecked(false);
+				api->on_event(OBS_FRONTEND_EVENT_BROADCAST_START_FAILED);
```

Comment:
This will wait until the user confirms the OBSMessageBox dialog before firing the event which is probably not good for an API use case. It might be better to fire this earlier.

Reply:
very good point; i have pushed the change.